### ISSUE TITLE
Platform module functions throw exceptions

### ISSUE THREAD

**Issue by cclauss** [brython-dev/brython#172]
At the Brython interactive prompt, the majority of the platform module functions throw exceptions.

```python
import platform
for func in (platform.architecture,    platform.machine,        platform.node,
             platform.platform,        platform.processor,      platform.python_build,
             platform.python_compiler, platform.python_branch,  platform.python_implementation,
             platform.python_revision, platform.python_version, platform.python_version_tuple,
             platform.release,         platform.system,         platform.version,
             platform.uname):
    print('{}.{:<23} = {}'.format(func.__module__, func.__name__ + '()', func()))
```

**Comment by kikocorreoso** [comment]
Hi @earney,

I think most of the functionality could be obtained using `window.navigator`. If you are busy I can be on charge of this.

**Comment by kikocorreoso** [comment]
I started to look at this to try to implement some functionality. The CPython platform module has the following functionality:

```>>> dir(platform)```
```
['DEV_NULL',
 '_UNIXCONFDIR',
 '__builtins__',
 '__cached__',
 '__copyright__',
 '__doc__',
 '__file__',
 '__loader__',
 '__name__',
 '__package__',
 '__spec__',
 '__version__',
 '_default_architecture',
 '_dist_try_harder',
 '_follow_symlinks',
 '_ironpython26_sys_version_pars
 '_ironpython_sys_version_parser
 '_java_getprop',
 '_libc_search',
 '_lsb_release_version',
 '_mac_ver_xml',
 '_node',
 '_norm_version',
 '_parse_release_file',
 '_platform',
 '_platform_cache',
 '_pypy_sys_version_parser',
 '_release_filename',
 '_release_version',
 '_supported_dists',
 '_sys_version',
 '_sys_version_cache',
 '_sys_version_parser',
 '_syscmd_file',
 '_syscmd_uname',
 '_syscmd_ver',
 '_uname_cache',
 '_ver_output',
 '_win32_getvalue',
 'architecture',
 'collections',
 'dist',
 'java_ver',
 'libc_ver',
 'linux_distribution',
 'mac_ver',
 'machine',
 'node',
 'os',
 'platform',
 'popen',
 'processor',
 'python_branch',
 'python_build',
 'python_compiler',
 'python_implementation',
 'python_revision',
 'python_version',
 'python_version_tuple',
 're',
 'release',
 'subprocess',
 'sys',
 'system',
 'system_alias',
 'uname',
 'uname_result',
 'version',
 'win32_ver']```

For security reasons, most of the information can't be accessed from a browser. I will start to work on this in order to obtain most of the useful functionality and when not possible an `Exception` will raise.

Is this ok? Other ideas?

**Comment by jinankjain** [comment]
Hi I would like to work on this bug could someone help me out?

**Comment by earney** [comment]
Jinank,

I can help you.

Billy

**Comment by jinankjain** [comment]
I am a newbie so how do i start of with this?

**Comment by earney** [comment]
I would look at the .js module files for brython located at www/src/libs.
If you are not comfortable with javascript, let me know and I'll find some
modules under www/src/Lib that you can use as modules.

Billy

**Comment by olemis** [comment]
@kikocorreoso please do not raise exceptions , that indeed breaks the contract of this module object .

**Comment by PierreQuentel** [comment]
This was fixed in commit 234ac281df40c9fce5b7afe8a735aa5f7ee919f4
